modern/src/MainPage.js

```diff
@@ -105,6 +108,8 @@ const MainPage = () => {
     setCollapsed(!collapsed);
   };
 
+  useEffect(() => dispatch(devicesActions.filterByName(deviceName.trim())), [deviceName]);
```

Comment:
It would be nice to filter by other parameters as well. At least device id.

Reply:
Do you mean using the same input text?
![image](https://user-images.githubusercontent.com/17521730/153064404-b7a91f8d-67a0-4d07-ae07-c1776c329cc1.png)

Since it has only one text input, a way to do that would be comparing also de the device id. Something like this:
```
filterByNameOrDeviceId(state, action) {
      return {
        ...state,
        filteredItems: action.payload
          ? Object.values(state.items).filter((item) => item.name?.toLowerCase().includes(action.payload?.toLowerCase()) || item.deviceId === action.payload)
          : state.items,
      };
    },
```
It's OK?

Reply:
Yeah, search in both. Don't do equals.